projects/schematics/src/shared/constants.ts

```diff
@@ -849,6 +849,8 @@ export const DEFAULT_USER_GROUP_ROUTING_CONFIG =
   'defaultUserGroupRoutingConfig';
 export const DEFAULT_USER_ROUTING_CONFIG = 'defaultUserRoutingConfig';
 
+export const CUSTOMER_TICKETING_SCSS_FILE_NAME = 'customer-ticketing.scss';
```

Comment:
anything in this folder will be moved and refactored. Similarly, to the schematics folder in the feature-lib of customer-ticketing

There's an ongoing work that changes our schematics 'way' of creating things. Will need to wait to update this. But we can continue on for now